src/components/ValueChange/ValueChange.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
```

Comment:
@Partysun `ValueChange` was created now as a separate component for feature use. It has a much more versatile interface, which is not compatible with current `PercentChanges`.  `PercentChanges` is used in the components with an old design which are targets of large refactoring. Also, it's much better to keep PR changes related to the only specific idea, which in our case is 'trends'

Reply:
@DmitriVanGuard Upgrade PercentChange, remove ValueChange. It's duplicated by logic. We kill 2 problems. First, remove font-awesome usage in PercentChange. You can add old/new value as options. It's easiest and bad practice to do any new staff from scratch.
 
Keep PR changes related to one specific idea means not to make new classes for any your idea. If you need such thing as ValueChange, better to make separate PR. It's one of the reasons why we have big PRs. When you are doing a big feature, you make modular things in this pr.

Reply:
@Partysun Please, see what the `PercentChange`. It has completely different logic which is also implied by its name. `PercentChange` always appends a percent sign a the end of the value and have weir interface. If we make percent sign optional, then suprise-suprise its becomes `ValueChange` 😄. `ValueChange` was created because `PercentChange` could not be used in the design implementation. 

Speaking of bad practices: Adding props whenever we want to **IS** a bad practice which inflates the logical borders of the components. As I mentioned before, `PercentChange` could be composed by `ValueChange` which is **recommended** pattern by the [core React team](https://reactjs.org/docs/composition-vs-inheritance.html).

Reply:
@DmitriVanGuard About upgrade PercentChange I mean to make anything inside this class and optional rename this module. If we make one more basic class than we have to refactor old class and use composition but make such a new class without refactoring old, you produce 2 branches. Make separate PR and refactor all usages of old class with new changes/basic class. 
Add story for this class.

Reply:
Sure, that was exactly one of my points! The old component will be refactored with an update of all dependencies in separate PR 👍